api/DOMRectReadOnly.json

```diff
@@ -5,42 +5,130 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/DOMRectReadOnly",
         "spec_url": "https://drafts.fxtf.org/geometry/#DOMRect",
         "support": {
-          "chrome": {
-            "version_added": "61"
-          },
-          "chrome_android": {
-            "version_added": "61"
-          },
-          "edge": {
-            "version_added": "79"
-          },
-          "firefox": {
-            "version_added": "31"
-          },
-          "firefox_android": {
-            "version_added": "31"
-          },
+          "chrome": [
+            {
+              "version_added": "61"
+            },
+            {
+              "alternative_name": "ClientRect",
+              "version_added": "2",
+              "version_removed": "61"
+            }
+          ],
+          "chrome_android": [
+            {
+              "version_added": "61"
+            },
+            {
+              "alternative_name": "ClientRect",
+              "version_added": "18",
+              "version_removed": "61"
+            }
+          ],
+          "edge": [
+            {
+              "version_added": "79"
+            },
+            {
+              "alternative_name": "ClientRect",
+              "version_added": "12",
+              "version_removed": "79"
+            }
+          ],
+          "firefox": [
+            {
+              "version_added": "31"
+            },
+            {
+              "alternative_name": "DOMRect",
```

Comment:
I had to add this to not have a hole in the support for Firefox. It looks like Firefox first just renamed `ClientRect` to `DOMRect` and then split it into `DOMRect` and `DOMRectReadOnly` with all the extra bits. It's a bit weird to say that `DOMRect` was an alias of `DOMRectReadOnly`, but only because of their current relationship. The `DOMRect` in Firefox 27-30 was just like `ClientRect` in 3-26 AFAICT.

I'm not sure if some note would help explain this?

Reply:
Yeah, I'm not sure there's a note to be written that doesn't exaggerate what the looks-weird-now-but-made-sense-then quality of the data. Let's leave this as-is. 👍